drivers/video/ov5640.c

```diff
@@ -6,6 +6,8 @@
 
 #define DT_DRV_COMPAT ovti_ov5640
 
+#include <math.h>
```

Comment:
maybe these features should be optional? maybe not every platform can afford to link in math?

Reply:
I saw many drivers including math.h without constraints, can you please explain why it might be necessary?